src/test/java/org/alfresco/util/CachingDateFormatTest.java

```diff
@@ -0,0 +1,43 @@
+package org.alfresco.util;
```

Comment:
I think we usually have copyright headers - even for test files.

Reply:
Done!